views/user/edit.ejs

```diff
@@ -33,13 +33,15 @@
           </div>
         </div>
         <div class="form-group row">
-          <div class="col-sm-10 offset-sm-2">
-              <input type="submit" class="btn btn-primary" value="Save Changes">
-              <input type="reset" class="btn btn-primary color-cancel" value="Cancel">
-              <a href="<%=`/user/${user._id}` %>">
-                <input class="btn btn-primary color-cancel profile-buttons" value="<< Back"></button>
-               </a>
-              
+          <div class="col-sm-4 offset-sm-2">
+            <input type="submit" class="btn btn-outline-success mr-2" value="Save changes">
+            <input type="reset" class="btn btn-outline-primary color-cancel" value="Cancel">
+
+            <a href="<%=`/user/${user._id}` %>">
+              <input class="btn btn-outline-primary color-info profile-buttons"
+                value="« Back to profile"></button>
```

Comment:
![grafik](https://user-images.githubusercontent.com/38247392/103535024-43855200-4e90-11eb-9a36-797afafba01e.png)
this has a weird margin, you can delete "profile-buttons" class to remove that. Also i would only have CANCEL or Back to profile, both would be duplicate.
Then this "save" button: in "styles.css" you could change the green color of this btn-outline-success button to a color which fits our website better XD maybe a softer green

Reply:
@lyrated cancel is for the form to delete all the new stuff entered, just clear the fields. Back is redirect, they are not dublicates

Reply:
hmm that's true, well we can leave that then :P